src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_api_server_authorized_ip_ranges.yaml

```diff
@@ -14,10 +14,9 @@ interactions:
       - --resource-group --name --location --dns-name-prefix --node-count --ssh-key-value
         --load-balancer-sku --api-server-authorized-ip-ranges
       User-Agent:
-      - AZURECLI/2.49.0 azsdk-python-azure-mgmt-containerservice/24.0.0 Python/3.8.10
-        (Linux-5.15.0-1039-azure-x86_64-with-glibc2.29)
+      - AZURECLI/2.57.0 azsdk-python-core/1.28.0 Python/3.11.5 (macOS-14.3-x86_64-i386-64bit)
```

Comment:
probably you don't need to update this file

Reply:
removed these changes